SofaKernel/modules/SofaBaseTopology/src/SofaBaseTopology/MeshTopology.cpp

```diff
@@ -21,6 +21,8 @@
 ******************************************************************************/
 #include <SofaBaseTopology/MeshTopology.h>
 
+#include <sofa/core/topology/Topology.h>
+#include <SofaBaseTopology/HexahedronSetTopologyContainer.h>
```

Comment:
this one is not logical, MeshTopology and TopologyContainer are apposite implementation of topology.

Reply:
alright, I'll put the hexahedron variables in Topology.h